Modules/CastStatus.cs

```diff
@@ -70,19 +70,19 @@ public CastTrackInfo(SNOPower power, Predicate<CastTrackInfo> success)
         /// Stone of recall spell (town portal)
         /// </summary>
         public CastTrackInfo StoneOfRecall = new CastTrackInfo(SNOPower.UseStoneOfRecall,
-            i => i.DurationMs > 3900 || ZetaDia.Globals.IsLoadingWorld || ZetaDia.Globals.WorldSnoId != i.WorldId);
+            i => i.DurationMs > 3750 || ZetaDia.Globals.IsLoadingWorld || ZetaDia.Globals.WorldSnoId != i.WorldId);

```

Comment:
What is the purpose of this magic number? Can you move it to a const variable with a proper name for the sake of future maintenance.